### PR TITLE
Update for polymerelements.

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,13 +10,16 @@
 <html>
 <head>
 
-  <script src="../webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="../core-component-page/core-component-page.html">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
```
Comment (morethanreal) [inline index.html]: need a `title` for this page?

File: demo/index.html [hunk 2]
```diff
@@ -16,12 +16,12 @@
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-capable" content="yes">
```
Comment (morethanreal) [inline demo/index.html]: need the word `demo` in the `title`?

**Comment by morethanreal** [comment]
lgtm w/ nit

File: index.html (hunk 1 above)
Comment (cdata) [inline index.html]: Resolved.

File: demo/index.html (hunk 2 above)
Comment (cdata) [inline demo/index.html]: Resolved.

**Pull request merged**